audits/2017-03.md

```diff
@@ -20,7 +20,7 @@ The git commit hash we evaluated is:
 
 # Disclaimer
```

Comment:
I've been against changing archived audits but if consensus is we should change it, that's fine.

Reply:
I don't think there is consensus. Just grouping everything under one PR.

Reply:
I agree we shouldn't change the audits. However, in this case I'm starting to lean towards changing it just because it's the most pragmatic decision to avoid having more bot PRs in the future. We've already closed and discussed this a couple of times.